src/app/containers/StoryPromo/index.jsx

```diff
@@ -1,18 +1,87 @@
 import React, { Fragment, useContext } from 'react';
+import moment from 'moment';
```

Comment:
You can import moment from `moment-timezone` here, means we dont have to have 2 moment packages in our package json :)